app/javascript/react/components/Reviews/ReviewTile.js

```diff
@@ -1,15 +1,15 @@
 import React from "react"
 
-const ReviewTile = ({ title, rating, body, date }) => {
-  
+const ReviewTile = ({ title, rating, body, date, user_id }) => {
   return (
     <ul>
       <h3>{title}</h3>
       <h5>Rating: {rating}</h5>
       <p>{body}</p>
       <p>Posted at: {date}</p>
+      <p>User Id: {user_id}</p>
```

Comment:
why...in the world would we need to see their user_id? username or even email I can understand, but id?